### ISSUE TITLE
Not able to deselect single star after selecting

### ISSUE THREAD

**Issue by bonjourrohit89** [kabuki5/CustomRatingBar#1]
Hi, I am using this library for custom rating bar but I am facing one problem, when I am selecting only one star in rating bar, Now I want to deselect that one star but I am not able to deselect same. Can you please tell me how can I deselect that single star?

Thanks in Advance

**Comment by kabuki5** [comment]
Hi,

Thanks for using the library. At this moment is not possible to revert the status to non selected rating, it's not a normal behavior at most rating bars. 
But I'm going to modify it to achieve your suggestion, I think is very interesting. 
I think, maybe a great solution could be when tap again at the first star, clear the selection and go back to non selected status. 

What do you think about it?

**Comment by bonjourrohit89** [comment]
Yes, It should be same as normal rating bar behaviour mean if user select one star and deselect same star then it should be deselect.
